public/src/main/java/com/revenuecat/purchases/models/StoreTransaction.kt

```diff
@@ -82,7 +82,18 @@ data class StoreTransaction(
      * Amazon's marketplace. Null for Google
      */
     val marketplace: String?
-) : Parcelable
+) : Parcelable {
+
+    /**
+     * Skus associated with the transaction
+     */
+    @Deprecated(
+        "Replaced with productIds",
+        ReplaceWith("productIds")
+    )
+    val skus: List<String> = productIds
```

Comment:
I think we can just set the getter? Like:
```suggestion
    val skus: List<String>
        get() = productIds
```
That way it doesn't create a backing field and just uses the getter.

Reply:
actually, looking at this again, i wonder if it's ok to return productIds...should i just be getting skus from the original object, so there's no behavior change at all?

Reply:
adding todo to the [StoreProduct PR](https://github.com/RevenueCat/purchases-android/pull/680) to follow up, so i can merge this one